lib/src/models/data/schedule.dart

```diff
@@ -73,6 +74,8 @@ class ScheduleModel extends Model {
 					day == null ? null : Day.fromJson(day)
 			]
 		];
+		Schedule.defaults = Map.from(await CloudCalendar.schedules.
```

Comment:
You can't use the actual class Cloud Calendar here, you need to use an instance (look up static vs instance methods if you're confused). Get the database from:

```dart
Services.instance.database.schedule
```

And call default schedules on that. Of course, you'll need to implement that too, so look for how the database downloads the `schedules` and try to copy that.